modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateLocalServiceTest.java

```diff
@@ -109,6 +118,55 @@ public void testAddNotificationTemplate() throws Exception {
 			notificationTemplate);
 	}
 
+	@Test
+	public void testAddNotificationTemplateError() throws Exception {
+		User user = TestPropsValues.getUser();
+
+		AssertUtils.assertFailure(
+			NotificationRecipientSettingNameException.class,
+			"Notification recipient setting name is invalid",
+			() -> _notificationTemplateLocalService.addNotificationTemplate(
+				NotificationTemplateUtil.createNotificationContext(
+					user, StringUtil.randomString(255),
+					NotificationConstants.TYPE_USER_NOTIFICATION)));
+
+		AssertUtils.assertFailure(
```

Comment:
The `NotificationConstants.TYPE_EMAIL` test case should come before the `NotificationConstants.TYPE_USER_NOTIFICATION` test case